src/Parser/RetributionPaladin/Modules/PaladinCore/Retribution.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import getDamageBonus from '../PaladinCore/GetDamageBonus';
+
+const RETRIBUTION_DAMAGE_BONUS = 0.2;
+
+class Retribution extends Module {
+	static dependencies = {
+		combatants: Combatants,
+	};
+
+	bonusDmg = 0;
+
+	on_byPlayer_damage(event){
+		if(!this.combatants.selected.hasBuff(SPELLS.RETRIBUTION_BUFF.id))
+			return;
+		this.bonusDmg += getDamageBonus(event, RETRIBUTION_DAMAGE_BONUS);
+	}
+
```

Comment:
Missing the suggestion that without Retribution proccing you should have someone die for your rankings 😅